src/outfit.ts

```diff
@@ -217,6 +221,14 @@ export class Outfit {
       }
     }
 
+    for (const mode in this.modes) {
+      if (haveEquipped(modeables[mode as keyof Modes])) {
+        const cmd = this.modes[mode as keyof Modes];
+        const args = Array.isArray(cmd) ? cmd.join(" ") : cmd;
+        cliExecute(`${mode} ${args}`);
```

Comment:
Maybe only change the mode if it doesn't match the current setting for the modeable? Otherwise this can lead to a lot of cli spam.